app/controllers/reports/regions_controller.rb

```diff
@@ -163,11 +168,19 @@ def find_region
         scope = current_admin.accessible_facilities(:view_reports)
         FacilityDistrict.new(name: report_params[:id], scope: scope)
       when "district"
-        current_admin.accessible_district_regions(:view_reports).find_by!(slug: report_params[:id])
+        if region_reports_enabled?
+          current_admin.accessible_district_regions(:view_reports).find_by!(slug: report_params[:id])
+        else
+          current_admin.accessible_facility_groups(:view_reports).find_by!(slug: report_params[:id])
+        end
```

Comment:
here is the issue - we need to use the FG slug when region reports are off, because those are the links users will be following.